src/lib/utils/analytics.ts

```diff
@@ -86,14 +81,14 @@ function getQuoteMethod(trade: InterfaceTrade) {
 export const formatSwapQuoteReceivedEventProperties = (
   trade: InterfaceTrade,
   allowedSlippage: Percent,
-  swapQuoteReceivedDate: Date
+  swapQuoteLatency: number | undefined
```

Comment:
i think for these `number` metrics its always helpful to suffix it by the unit used, like `swapQuoteLatencyMs`